cas_implementations/src/eav/file.rs

```diff
@@ -90,6 +90,24 @@ impl EavFileStorage {
         Ok(())
     }
 
+    fn delete_file(
```

Comment:
Am I incorrect in thinking that the EAV was append only? 
@lucksus

Reply:
We already synced in heartbeat and currently implementing eavindexing before this can get in

Reply:
ah, I see. Sorry to interrupt!

Do you mean that you are currently switched to implementing indexing the EAV before proceeding further with this PR?

Reply:
No problem! Yes :) Deleting will simply be changing the hash to a new one which does not contain the data! Working on that right now